released/packages/coq-infotheo/coq-infotheo.0.0.5/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "reynald.affeldt@aist.go.jp"
+homepage: "https://github.com/affeldt-aist/infotheo"
+bug-reports: "https://github.com/affeldt-aist/infotheo/issues"
+dev-repo: "git+https://github.com/affeldt-aist/infotheo.git"
+license: "GPLv3"
+authors: [
+  "Reynald Affeldt"
+  "Manabu Hagiwara"
+  "Jonas Senizergues"
+  "Jacques Garrigue"
+  "Kazuhiko Sakaguchi"
+  "Taku Asai"
+  "Takafumi Saikawa"
+  "Naruomi Obata"
+  "Erik Martin-Dorel"
+  "Ryosuke Obi"
+  "Mitsuharu Yamamoto"
+]
+build: [
+  ["coq_makefile" "-f" "_CoqProject" "-o" "Makefile"]
+  [make]
```

Comment:
```suggestion
  [make "-j%{jobs}%"]
```
Is there any specific reason to disable parallelism here? 

I would also advice that the `coq_makefile` command is moved to a custom Makefile inside the project, but that would be for future releases.

Reply:
parallelism enabled (custom makefile deferred to later)